native-schema-registry/python/PyGsrSerDe/test_AvroSerDe.py

```diff
@@ -33,9 +32,51 @@ def test_serde(self):
         }
         parsed_schema = ser.parse_schema(schema)
 
+        complex_table = [{'name': 'Bob', 'friends': 42, 'age': 33, 'dominant hand': 'right',
+                          'allergies': ['apples', 'strawberries']},
+                         {'name': 'Jim', 'friends': 13, 'age': 69, 'dominant hand': 'right', 'allergies': []},
+                         {'name': 'Joe', 'friends': 86, 'age': 17, 'dominant hand': 'left',
+                          'allergies': ['bananas', 'strawberries']},
+                         {'name': 'Ted', 'friends': 23, 'age': 51, 'dominant hand': 'right', 'allergies': []}]
+
+        complex_schema = {
```

Comment:
In addition to one complex schema, I think it is also critical to have individual tests for each scenario so it will be more effective for identifying the issue.
For example, there are issues for schema contains array and map but not enum. Then it will be more straight forward if there are separate test cases for them.

Reply:
Added some individual test cases.